### PR TITLE
[mod_articles_latest]XML code style

### PR DESCRIPTION
**Pull request by wojsmol**
### Summary of Changes

Code style changes for mod_articles_latest
### Testing Instructions

code review

cc @JoomliC @andrepereiradasilva

### CODE REVIEW

File: modules/mod_articles_latest/mod_articles_latest.xml [hunk 1]
```diff
@@ -74,7 +75,7 @@
 					default="0"
 					label="MOD_LATEST_NEWS_FIELD_USER_LABEL"
```
Comment (andrepereiradasilva) [inline modules/mod_articles_latest/mod_articles_latest.xml]: move defaut down?

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on 6518f2bda4bef3f6bf740747f259a11cd418ad24<br /><br />on code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10574">issues.joomla.org/joomla-cms/10574</a>.</sub>

**Comment by JoomliC** [comment]
I have tested this item :white_check_mark: successfully on 6518f2bda4bef3f6bf740747f259a11cd418ad24<br /><br />On code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10574">issues.joomla.org/joomla-cms/10574</a>.</sub>

**Comment by brianteeman** [comment]
RTC <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10574">issues.joomla.org/joomla-cms/10574</a>.</sub>

**Pull request closed without merging**